lambda-api/outputs.tf

```diff
@@ -1,5 +1,3 @@
-
 output "api_url" {
-    value = "${data.aws_apigatewayv2_api.api_endpoint}"
+  value = data.aws_apigatewayv2_api.apigw
```

Comment:
This should be `data.aws_apigatewayv2_api.apigw.api_endpoint`

Reply:
@euforia - Updated.